.travis.yml

```diff
@@ -21,8 +21,8 @@ script:
  - pytest --cov pypistats
 
  # Static analysis
- - flake8 --statistics --count
- - if [[ $TRAVIS_PYTHON_VERSION == 3.6 ]]; then black --check --diff .; fi
+ - flake8 --statistics --count --verbose pypistats/ tests/
+ - if [[ $TRAVIS_PYTHON_VERSION == 3.6 ]]; then black --check --diff --verbose pypistats/ tests/; fi
```

Comment:
Is there a reason to specify these dirs and not run in `.`? We miss `setup.py` like this.

Also, let's leave out `--verbose` so the output is a cleaner.

Reply:
I didn't think about `setup.py`, sorry.

I put `--verbose` in case of referring Travis build logs via URL if anyone wants to contribute to show contributors what the problem is.

Reply:
Sorry, just read your PR message fully now, and it's done for speed. 

The CI took 3.13s to run Black with this PR:

https://travis-ci.org/hugovk/pypistats/jobs/447664623#L640

and 4.54s with master:

https://travis-ci.org/hugovk/pypistats/jobs/447539626#L580

I think that's fine.

Reply:
If there's a flake8 error, it lists them in the logs wihout `--verbose`.

And if Black errors, `--diff` shows what should be have been done.

I think the verbose stuff is probably a bit distracting.